BasePreparedModel.cpp

```diff
@@ -424,12 +426,145 @@ Return<V1_3::ErrorStatus> BasePreparedModel::execute_1_3(
 }
 
 Return<void> BasePreparedModel::executeFenced(
-    const V1_3::Request& request, const hidl_vec<hidl_handle>& waitFor, V1_2::MeasureTiming measure,
-    const V1_3::OptionalTimePoint& deadline,
+    const V1_3::Request& request1_3, const hidl_vec<hidl_handle>& waitFor,
+    V1_2::MeasureTiming measure, const V1_3::OptionalTimePoint& deadline,
     const V1_3::OptionalTimeoutDuration& loopTimeoutDuration,
     const V1_3::OptionalTimeoutDuration& duration, executeFenced_cb cb) {
     ALOGV("Entering %s", __func__);
-    // TODO: Add support
+
+    time_point driverStart, driverEnd, deviceStart, deviceEnd, driverAfterFence;
```

Comment:
Nit : https://google.github.io/styleguide/cppguide.html#Local_Variables, recommends to declare vars as we use. we can follow this going forward.